wsgi/odyssy/people/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.core.urlresolvers import reverse
+
+from django.db import models
+
+
+class People(models.Model):
+    class Meta:
+        ordering = ('name',)
+        verbose_name = 'People'
+        verbose_name_plural = 'People'
+
+    STATUS_CHOICES = (
+        ('faculty', 'Faculty'),
+        ('staff', 'Staff'),
+        ('visiting_faculty', 'Visiting Faculty'),
+    )
```

Comment:
sir I've updated it in `visiting faculty` with the post of Director as it is on our current website. If we have our own director we can directly add them in our staff's page with the post director